src/garage/experiment/snapshotter.py

```diff
@@ -41,21 +41,46 @@ def __init__(self,
 
     @property
     def snapshot_dir(self, ):
```

Comment:
why all the trailing commas?

Reply:
It should not be there. I will remove it.